src/Makefile

```diff
@@ -24,7 +24,7 @@ INSTALL ?= install
 
 soname=$(NAME).so.1
 minor=0
-micro=1
+micro=2
```

Comment:
I don't think we have to update the version number yet, as we haven't released the shared library yet.

Reply:
I haven't any experience with shared library versioning. I suppose there is some documentation available how it supposed to be done.